### ISSUE TITLE
Issue - but not to be resolved - how the app handles a different phone number

### ISSUE THREAD

**Issue by michiQ** [yadaguru/yadaguru-app#105]

**Comment by michiQ** [comment]
So this is one just to know - but I don't think we have to do anything about it really.  

If I have logged in on my phone - say its 555-555-5555.  Then I log out.

Then I get the login screen and I want to login using 777-777-7777.   Assuming I have the device for 777-777-7777 and get the confirm code and enter it, the app by passes the new user flow.  I think this is probably because the phone has a cookie that says that the new user flow has already ran on the device.

I actually don't think this is really a issue becuase the use case is someone using a number not connected to the device they are on.  

What's interesting  - and I never thought of this before - is that the app _will_ let you use yadaguru  on a device that is not linked to the login phone number, if that person can access the confirm code.

So just FYI for backlog purposes.

**Comment by bobderrico80** [comment]
Yes that is correct. A person can use the app on a desktop... the associated phone number is just to verify the identity of the user (like the 2nd factor in two-factor authentication).

There is indeed a local storage setting that prevents onboarding from happening on login if the user (identified by their phone number) has already gone through the login process.
